scripts/epa/airdata/README.md

```diff
@@ -24,19 +26,27 @@ and relies on the following StatisticalVariables:
 - Mean_Concentration_AirPollutant_PM10
 - Max_Concentration_AirPollutant_PM10
 - AirQualityIndex_AirPollutant_PM10
+- AirQualityIndex_AirPollutant (only for county/CSBA)
 
 ## Notes on the Data
-In the cleaned CSV, observations are provided on the site monitor level. For simplicity in the place mapping, we select a single monitor per site and pollutatn and provide the monitor id (POC) as airQualitySiteMonitor in the observation. So, a given observation is distinguished by observationDate, observationAbout (AirQualitySite), and measurementMethod (pollutant standard).
+Pollutant-specific metrics are provided on the site monitor level. For simplicity in the place mapping, we select a single monitor per site and pollutant and provide the monitor id (POC) as airQualitySiteMonitor in the observation. So, a given observation is distinguished by observationDate, observationAbout (AirQualitySite), and measurementMethod (pollutant standard).
+
+County/CBSA level AQI metrics include the defining site and pollutant, which are included in the StatVarObservation.
```

Comment:
I guess I'm confused what a "defining pollutant" is and why it is not in the StatVar.

Reply:
this refers to the pollutant that is used to get the aggregate aqi measurement (I believe it's just the max out of all of the individual pollutant aqis)